gulpfile.js

```diff
@@ -1,16 +1,39 @@
-var elixir = require('laravel-elixir');
-
-/*
- |--------------------------------------------------------------------------
- | Elixir Asset Management
- |--------------------------------------------------------------------------
- |
- | Elixir provides a clean, fluent API for defining some basic Gulp tasks
- | for your Laravel application. By default, we are compiling the Sass
- | file for our application, as well as publishing vendor resources.
- |
+var gulp = require('gulp');
+
+/**
+ * Copy any needed files.
+ *
+ * Do a 'gulp copyfiles' after bower updates
  */
+gulp.task("copyfiles", function() {
```

Comment:
I think it's not very good to copy file. Combining many files to only 1 file is better. Please use Larave elixir to do that. 
https://laravel.com/docs/5.2/elixir